packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards.scss

```diff
@@ -66,6 +62,11 @@
       }
     }
 
+    .#{$prefix}--card-link__title,
+    .#{$prefix}--card-link__content {
+      width: 90%;
+    }
```

Comment:
There is also a grid inside of the card that I'm personally choosing to ignore for the moment because I don't think it's correct. (if we follow the grid I believe it would look something like this...)

```scss
{
  width: 67.5%; // 90% of 75% or (6, 8)
  max-width: carbon--mini-unit(80);
}
```

@oliviaflory @wonilsuhibm any thoughts on the grid inside of the card?

![image](https://user-images.githubusercontent.com/3793636/74270111-515a4380-4cd0-11ea-8e40-fe0492d27354.png)